docs/modules/reference/pages/service-assurance/monitors/JDBCQueryMonitor.adoc

```diff
@@ -174,18 +183,28 @@ Note that you must include the `monitor` section in your definition.
 [source, xml]
 ----
 <service name="MariaDB-Galera" interval="300000" user-defined="false" status="on">
-  <parameter key="driver" value="org.mariadb.jdbc.Driver"/>
-  <parameter key="user" value="opennms"/>
-  <parameter key="password" value="********"/>
-  <parameter key="url" value="jdbc:mysql://OPENNMS_JDBC_HOSTNAME"/>
-  <parameter key="query" value="SELECT VARIABLE_VALUE FROM INFORMATION_SCHEMA.GLOBAL_STATUS WHERE VARIABLE_NAME = 'wsrep_cluster_status'"/>
-  <parameter key="column" value="VARIABLE_VALUE"/>
-  <parameter key="action" value="compare_string"/>
-  <parameter key="operator" value="="/>
-  <parameter key="operand" value="Primary"/>
-  <parameter key="message" value="Galera Node is not in primary component"/>
+  <parameter key="driver" value="org.mariadb.jdbc.Driver"/> <1>
+  <parameter key="user" value="opennms"/> <2>
+  <parameter key="password" value="********"/> <3>
+  <parameter key="url" value="jdbc:mysql://OPENNMS_JDBC_HOSTNAME"/> <4>
+  <parameter key="query" value="SELECT VARIABLE_VALUE FROM INFORMATION_SCHEMA.GLOBAL_STATUS WHERE VARIABLE_NAME = 'wsrep_cluster_status'"/> <5>
+  <parameter key="column" value="VARIABLE_VALUE"/> <6>
+  <parameter key="action" value="compare_string"/> <7>
+  <parameter key="operator" value="="/> <8>
+  <parameter key="operand" value="Primary"/> <9>
+  <parameter key="message" value="Galera Node is not in primary component"/> <10>
 </service>
 
-<monitor service="MariaDB-Galera" class-name="org.opennms.netmgt.poller.monitors.JDBCQueryMonitor" /> <1>
+<monitor service="MariaDB-Galera" class-name="org.opennms.netmgt.poller.monitors.JDBCQueryMonitor" /> <11>
 ----
-<1> Required *monitor* section.
\ No newline at end of file
+<1> JDBC driver class to use.
+<2> Database user
+<3> Database password
+<4> JDBC URL to connect to.
+<5> The SQL query to run.
+<6> The result column to evaluate against when using compare_string method.
```

Comment:
Markdown isn't happy with the adoc format, the markdown gets semi lost when rendered [no gray background]. This will need to be resolved likely a different way.